### PR TITLE
[release-4.15] OCPBUGS-28703: rollout metrics-server on cert rotations

### PR DESCRIPTION
**Pull request by slashpai**
This commit ensures that the metrics-server's pods are redeployed when the Kubelet Serving CA/Metrics Client Cert or Serving CA Cert are rotated.

* [ ] I added CHANGELOG entry for this change.
* [x] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

**Comment by slashpai** [comment]
cc: @machine424

File: pkg/tasks/metricsserver.go [hunk 1]
```diff
@@ -3,11 +3,13 @@ package tasks
 import (
 	"context"
 
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+
 	"github.com/openshift/cluster-monitoring-operator/pkg/client"
 	"github.com/openshift/cluster-monitoring-operator/pkg/manifests"
 	"github.com/pkg/errors"
```
Comment (simonpasquier) [inline pkg/tasks/metricsserver.go:12]: we need to move this up.

**Review by simonpasquier — COMMENTED** [review]
the imports order in `pkg/tasks/metricsserver.go` is still incorrect.

**Pull request merged**